packages/api/src/external/hie/models/hie-directory-view.ts

```diff
@@ -0,0 +1,59 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { BaseModel, ModelSetup } from "../../../models/_default";
+
+export class HIEDirectoryEntryViewModel extends BaseModel<HIEDirectoryEntryViewModel> {
+  static NAME = "hie_directory_view";
+
+  declare oid: string;
```

Comment:
Don't we need `id` in addition to `oid` here?

Reply:
BTW, why the `oid` column if we're mapping the underlying table/view's `id` column to both `id` and `oid`?

Reply:
i generally prefer to always reserve id for a uuid for the entity. If there is another field that can also be used for a pk (like oid) that's nice, but i will still have a UUID typed id column. And conversely, if a field has any semantic significance, it gets its own column.

I initially had a uuid but i realized we dont follow this pattern in the CQ directory table, and i didn't want to mix uuids and oids for the 'id' column in the view, so I opted to just have an id column so the view works fine, and then an oid column.

Reply:
I also like the pattern to have the ID to be a UUID unrelated to any business logic.

It's just that, in this case, since we're already using the OID as the ID on both tables, I don't see any reason to have two columns for the same thing - the values are exactly the same, it just adds complexity (I see our future selves spending cycles wondering why is that).

Also, this made me think about something important: for pagination, we need individual IDs. And I'm not sure if we can't have the same OID on both directories - which would make pagination non-deterministic.

This could be a reason to keep the ID, but actually change the underlying tables so we have automatically generated UUIDs there.

Reply:
addressed - see https://metriport.slack.com/archives/C04CXLSAF7V/p1738197379959929?thread_ts=1737587162.651939&cid=C04CXLSAF7V